packages/typespec-ts/test/nonBrandedIntegration/modular/generated/models/usage/tspconfig.yaml

```diff
@@ -0,0 +1,15 @@
+emit:
```

Comment:
why do we need this file?

Reply:
This file is not generated, this is the config file for generation and the input of `tsp compile`. We enable this in integration so that we could have case-level controls for different options which could help us verify the combination of various setting in integration.

Reply:
Remove the azureSDKforJS config